enhancements/monitoring/monitoring-stack-operator.md

```diff
@@ -255,13 +260,21 @@ During the Monitoring Enablement Working Group meetings, we received feedback fr
 Moreover, Platform SRE expressed concerns that a shared Alertmanager configuration could lead to conflicts or misconfiguration that can impact alerting for every managed tenant running in the cluster.
 
 Based on these discussions, we suggest that a dedicated Alertmanager instance is deployed with each `MonitoringStack` CR in the same namespace as the CR itself.
-We can also take advantage of the recently added <code>[AlertmanagerConfig](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#alertmanagerconfig)</code> and allow MTO/MTSRE to create their own routing configuration in a centrally deployed Alertmanager instance.
+In next iteration we can focus on sharing AM across MonitoringStacks. We will take advantage of the recently added <code>[AlertmanagerConfig](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#alertmanagerconfig)</code> and allow MTO/MTSRE to create their own routing configuration in a centrally deployed Alertmanager instance.
+
+### Integrating with Console
+
+After discussions with console team we would like to avoid changing UX and add notion of data sources. Since OCS wants to integrate with OCP Console, we need to make sure Monitoring Stack addons can (opt-in) expose data to Platform Monitoring's Thanos Querier that is connected to Console.
+
+For this we added feature request for CMO recorded in [MON-1954](https://issues.redhat.com/browse/MON-1954). When implemented we will be able to deploy following architecture:
```

Comment:
this might require more discussions at the implementation level but they can happen after this enhancement is accepted.

Reply:
👍🏽